src/main/play-doc/general/ReleaseNotes.md

```diff
@@ -18,6 +19,16 @@ The new licensing mechanism permits a single ConductR Agent to be used absolutel
 
 Please visit the [migration guide](MigrationGuide#Production_Suite_Licensing) for additional instructions.
 
+### External Service Registration
+
+Since 2.0, ConductR has supported a DNS SRV record fallback mechanism for service discovery i.e. if a service cannot be discovered as being hosted by ConductR, then DNS SRV lookups can be performed in order to resolve the service's location. This was disabled for standalone mode, but enabled for DC/OS installation so that Elasticsearch lookups can utilize the DC/OS hosted service.
+
+As of 2.1, ConductR replaces the 2.0 mechanism with something considerably more flexible. Any service name may be registered via ConductR's configuration to exist as an external service, and one of: an explicit IP and port; DNS and port; or DNS SRV host may be supplied. Furthermore any number of these can be mapped to a service name and also resolved with user credentials (http basic auth at this point), and a root path.
```

Comment:
Also, it is possible to specify a set of URIs and not only one? The documentation does not point that out clearly enough.